relayer/relayer/src/listeners/eth.rs

```diff
@@ -154,26 +154,44 @@ impl EthereumListener {
     }
 }
 
+#[cfg(feature = "evm")]
```

Comment:
Ethereum is also evm :) Maybe call it differently? layer2, l2, rollup, arbitrum?

Reply:
`relayer-eth` and `relayer-l2` works for me (names of artifacts)
so cargo feature name would be just `l2` in such case

Reply:
ok, but lets do it not here. I have created https://cardinal-cryptography.atlassian.net/browse/MOST-95